### PR TITLE
Update Algorithm Spinal Tap Case

### PR DESCRIPTION
**Pull request by dakshshah96**
Refer #1094

### CODE REVIEW

File: Algorithm-Spinal-Tap-Case.md [hunk 1]
```diff
@@ -1,30 +1,43 @@
 # Algorithm Spinal Tap Case
 
-### Problem Explanation:
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](FreeCodeCamp-Get-Help) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :pencil:
 
-- Convert the given string to an all lowercase sentence joined by dashes.
+### :checkered_flag: Problem Explanation:
 
-## Hint: 1
+Convert the given string to an all lowercase sentence with words joined by dashes.
```
Comment (atjonathan) [inline Algorithm-Spinal-Tap-Case.md]: ~~Convert the given string to an all lowercase sentence with words joined by dashes.~~ Convert the given string to a lowercase sentence with words joined by dashes.

**Comment by atjonathan** [comment]
### **LGTM :+1:**

**Comment by dakshshah96** [comment]
@atjonathan Correction made 👍

File: Algorithm-Spinal-Tap-Case.md [hunk 2]
```diff
@@ -1,30 +1,43 @@
...
+- [String global object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)
+- [JS Regex Resources](JS-Regex-Resources)
+- [JS String Prototype Replace](JS-String-Prototype-Replace)
+- [JS String Prototype ToLowerCase](JS-String-Prototype-ToLowerCase)
 
-- You will also have to make everything lowercase.
+## :speech_balloon: Hint: 1
 
-## Hint: 3
+Create a regex for all white spaces and underscores.
```
Comment (atjonathan) [inline Algorithm-Spinal-Tap-Case.md]: ~~regex~~ regular expression

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks for contributing :tada:

File: Algorithm-Spinal-Tap-Case.md (hunk 2 above)
Comment (dakshshah96) [inline Algorithm-Spinal-Tap-Case.md]: Ah, thought about it and figured out since it was used everywhere, didn't need to change it. Done 👍

**Pull request merged**